src/sentry/tasks/assemble.py

```diff
@@ -297,7 +297,7 @@ def _remove_duplicate_artifact_bundles(org_id: int, ids: List[int]):
 
 
 def _bind_or_create_artifact_bundle(
-    bundle_id: uuid,
+    bundle_id: str,
```

Comment:
Couldn't this be `None` in addition to `str`?

Reply:
true, good catch